extensions/mssql/src/mssql.d.ts

```diff
@@ -875,6 +875,19 @@ declare module 'mssql' {
 
 	// Object Management - Begin.
 	export namespace ObjectManagement {
+
+		/**
+		 * Object types.
+		 */
+		export enum NodeType {
```

Comment:
This needs to be a const enum, we can't have definitions in typings files. 
```suggestion
		export const enum NodeType {
```